From: Sven Roederer <freifunk@it-solutions.geroedel.de>
Date: Thu, 13 May 2021 19:31:18 +0200
Subject: mod-freifunk: hotfix for "https://github.com/freifunk/openwrt-packages/issues/33"

* don't drop root priviledges make page work again, but it's not a real fix of the issue

diff --git a/modules/luci-mod-freifunk/luasrc/controller/freifunk/freifunk.lua b/modules/luci-mod-freifunk/luasrc/controller/freifunk/freifunk.lua
index f35fa60dd34fdbb3d0619061fc9051cb12f8537d..8379513336f3b11b95ad504d3a2bef0650d2a531 100644
--- a/modules/luci-mod-freifunk/luasrc/controller/freifunk/freifunk.lua
+++ b/modules/luci-mod-freifunk/luasrc/controller/freifunk/freifunk.lua
@@ -18,8 +18,9 @@ function index()
 	page.title    = _("Freifunk")
 	page.target   = alias("freifunk", "index")
 	page.order    = 5
-	page.setuser  = "nobody"
-	page.setgroup = "nogroup"
+-- TODO: fix issue #33 and drop privileges again
+--	page.setuser  = "nobody"
+--	page.setgroup = "nogroup"
 	page.i18n     = "freifunk"
 	page.index    = true
 
